services/QuillLMS/Gemfile

```diff
@@ -169,6 +170,7 @@ group :production, :staging do
   gem 'rails-autoscale-sidekiq'
   gem 'lograge'
   gem "matrix", '~> 0.4.2'
+  gem 'wkhtmltopdf-heroku', '2.12.6.1.pre.jammy'
```

Comment:
It's slightly less than ideal that there's a different gem for prod/staging vs. test/development. But if that's what we have to do for this specific use-case, so be it.

Reply:
I don't really like the asymmetry either.